changes/14464-add-kolide-tables

```diff
@@ -0,0 +1,29 @@
+- added tables to the fleetd extension: 
+  - airport_util
+  - app_icons
+  - apple_silicon_security_policy
+  - falconctl_options
+  - falcon_kernel_check
+  - cryptoinfo
+  - cryptsetup_status
+  - parse_plist
+  - parse_json
+  - parse_jsonl
+  - parse_xml
+  - parse_ini
+  - dev_table_tooling
+  - dsim_default_associations
+  - filevault_status
+  - firefox_preferences
+  - firmwarepasswd
+  - gsettings
+  - gsettings_metadata
+  - ioreg
+  - secedit
+  - system_profiler
```

Comment:
```suggestion
```

Noah: Let's cut this table. It also also has the strange/hard-to-use flat list data format. We can make it better later.